media/doc/3rd Party Files.txt

```diff
@@ -275,3 +275,9 @@ Path: sdk/tools/mkisofs/schilytools
 Used Version: schily-2017-09-07
 License: CDDL-1.0 (https://spdx.org/licenses/CDDL-1.0.html)
 URL: https://sourceforge.net/projects/schilytools/
+
+Title: AMD LibM for Windows
+Path: sdk/lib/crt/math/libm_sse2
+Used Version: ??? (Imported on 2022.12.01.)
```

Comment:
It's the last version. It is no longer maintained anyway. So you can use the head revision.

Reply:
Confirmed and updated.